src/components/AssetForm/components/NftSelect/index.tsx

```diff
@@ -209,7 +210,7 @@ export const NftSelect: React.FC<INftSelectProps> = ({
 
   const selectedCollectionImage = useMemo(
     () =>
-      selectedCollection ? nfts[selectedCollection?.id][0].imgUrl : undefined,
+      selectedCollection ? nfts[selectedCollection?.id][0]?.imgUrl : undefined,
```

Comment:
It can be simpler.
```
const selectedCollectionImage = useMemo(
    () => nfts[selectedCollection?.id]?.[0]?.imgUrl,
    [nfts, selectedCollection]
);
```

Reply:
I still need to check for `selectedCollection` otherwise typescript complains that the index for `nfts` cannot be undefined